package.json

```diff
@@ -44,7 +44,7 @@
     "zone.js": "^0.8.29"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "~0.13.0",
+    "@angular-devkit/build-angular": "^0.12.4",
```

Comment:
@drepram Why this change?

Reply:
It was an error that occurred on my machine. Take a look at https://stackoverflow.com/a/56624167/10103199.

Reply:
is this(^) error showed due to some change of yours?

Reply:
I never installed Angular CLI, I installed it and it didn't worked. Googled it and a lot of people had the same problem, shall I revert the change?

Reply:
yes please

Reply:
But then it wouldn't work my machine? Wouldn't that be bad assuming other people would have the same problem?

My machine is a very generic machine: `MacBook Pro (13-inch, 2017, Two Thunderbolt 3 ports`.

Reply:
But this isn't related to this PR, if you feel the issue is valid and it is occurring on the master branch too then please open a separate PR for that fix.

Reply:
Fair point.